x/subscription/spec/01_concepts.md

```diff
@@ -0,0 +1,21 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Subscription
+
+- Merchant create/stop plans on chain, the price is defined on specified token denomination, users should do nessessary token conversion in other ways, e.g. exchange.
```

Comment:
just nit -- perhaps for the consistency, "plan owners" instead of merchants (e.g. this could potentially be used for regular donations).
```suggestion
- Subscription plan owners create/stop plans on Chain, the plan prices are defined for specified token denominations, subscribers should do necessary token conversions in other ways, e.g. exchange.
```